PowerEditor/src/ScitillaComponent/FindReplaceDlg.cpp

```diff
@@ -3005,21 +3016,22 @@ void Progress::setPercent(unsigned percent, const TCHAR *fileName) const
 {
 	if (_hwnd)
 	{
-		::SendNotifyMessage(_hPBar, PBM_SETPOS, (WPARAM)percent, 0);
+		::PostMessage(_hPBar, PBM_SETPOS, (WPARAM)percent, 0);
 		::SendMessage(_hPText, WM_SETTEXT, 0, (LPARAM)fileName);
 	}
 }
 
 
-void Progress::close()
+void Progress::flushCallerUserInput() const
 {
-	if (_hwnd)
+	MSG msg;

```

Comment:
What's `msg` doing out here? Does it need to persist across loop iterations? (I doubt it?)